cost-optimization/gke-keda/cloud-pubsub/deployment/keda-pubsub-scaledobject.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_deployment_keda_pubsub_hpa_scaledobject_pubsub]
+# [START container_keda_pubsub_horizontal_pod_autoscaler]
```

Comment:
Some samples do have two region tags like this `gke_` and `container_` but it's more of a historical thing, and is not necessary. You could simply use one (`gke_`) and use that in your doc when you embed.

(Same comment in other files)

Reply:
Done, thanks. (Also I added tags in all files for completeness).